HACKING.md

```diff
@@ -1,17 +1,28 @@
 # Hacking on snapd
 
-Hacking on snapd is fun and straightforward. The code is extensively
-unit tested and we use the [spread](https://github.com/snapcore/spread)
-integration test framework for the integration/system level tests.
+Hacking on `snapd` is fun and straightforward. The code is extensively unit
+tested and we use the [spread](https://github.com/snapcore/spread) integration test framework for the
+integration/system level tests.
 
 ## Development
 
-### Supported Go versions
+### Supported Go version
 
-From snapd 2.52, snapd supports Go 1.13 and onwards. From snapd 2.38
-to 2.52, snapd requires Go 1.9+. Versions before 2.38 support Go 1.6+.
+Go 1.13 or later is required to build `snapd`.
```

Comment:
Just on this point: what would be the benefit for the person checking out the branch to have a version reference that does not apply to the checkout point? Would the snapd version not always be >= recent than v2.52? Sorry if I am missing something obvious, just trying to prevent adding stuff that will become dated soon.

Reply:
Thank you for the explanations and suggestions. I've gone with option two (please feel free to tell me otherwise), because the pre Go 1.11 version will require different build steps that will make build the instructions flow less elegant.